testutil/checkers_test.go

```diff
@@ -27,11 +27,16 @@
 package testutil
 
 import (
+	"path/filepath"
 	"reflect"
 	"runtime"
 	"testing"
 
 	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/strutil"
+	"io/ioutil"
```

Comment:
Can you group `io/ioutil` and `regexp` with the rest of stdlib imports?

Reply:
Ouch! bad emacs! :-/ fixing